crates/transpiler/src/equivalence.rs

```diff
@@ -149,6 +153,11 @@ impl Equivalence {
             && other_circuit.eq(&slf.getattr("circuit")?)?)
     }
 
+    #[getter]
+    fn get_circuit<'py>(slf: &'py Bound<'py, Self>) -> CircuitFromPython {
+        CircuitFromPython(slf.borrow().circuit.clone())
+    }
```

Comment:
Why not just?:

```suggestion
    #[getter]
    fn get_circuit(&self) -> CircuitFromPython {
        CircuitFromPython(self.circuit.clone())
    }
```
It's effectively the same thing but with less boilerplate right?